src/pages/components/breadcrumb/usage.mdx

```diff
@@ -13,11 +13,41 @@ The breadcrumb is a secondary navigation pattern that helps a user understand th
 <AnchorLinks>
 
 <AnchorLink>Overview</AnchorLink>
-<AnchorLink>Variations</AnchorLink>
+<AnchorLink>Live demo</AnchorLink>
+<AnchorLink>Formatting</AnchorLink>
+<AnchorLink>Content</AnchorLink>
+<AnchorLink>Behaviors</AnchorLink>
+<AnchorLink>Modifiers</AnchorLink>
+<AnchorLink>Related</AnchorLink>
 <AnchorLink>Feedback</AnchorLink>
 
 </AnchorLinks>
 
+## Overview
+
+Breadcrumbs show users their current location relative to the information architecture and enable them to quickly move up to a parent level or previous step.
+
+### When to use
+
+Breadcrumbs are effective in products and experiences that have a large amount of content organized in a hierarchy of more than two levels. They take up little space but still provide context for where in the navigation hierarchy the user is.
+
+### When not to use
+
+Breadcrumbs are always treated as secondary and should never entirely replace the primary navigation. They shouldn't be used for products that have single-level navigation because they create unnecessary clutter.
```

Comment:
```suggestion
Breadcrumbs are always treated as secondary and should never entirely replace the primary navigation. They shouldn't be used for products that have single level navigation because they create unnecessary clutter.
```